### PR TITLE
Creating basic API to get and update connection status, starting with getConnection call

### PR DESCRIPTION
**Pull request by ggenya132**
As per #24 requests

### CODE REVIEW

File: server/controllers/connection-status.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const {
+    queryDynamoByChannel,
+    updateConnectionStatus,
+} = require('../lib/dynamoDao');
+
+// TO BE USED LATER AS SOME KIND OF TYPESAFE ENUM VALIDATION
+const CONNECTION_STATUS = {
```
Comment (SaxyPandaBear) [inline server/controllers/connection-status.js:7]: We can try to throw in a typescript enum in here if you want. I'm not partial, though the last time I tried to use enums in Typescript, it ended poorly.

File: server/controllers/connection-status.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                } else {
+                    res.json({ item });
+                }
+            })
+            .catch((err) => {
+                console.log(err);
+                res.status(500).json({ err });
+            });
+    },
+    updateConnectionStatus(channelId, connectionStatus) {
```
Comment (SaxyPandaBear) [inline server/controllers/connection-status.js:32]: I don't like that the function name here is shadowed by the name in the dynamo lib. This causes ambiguity and harder to trace. Either the method name in this controller needs to change, or the one in the dynamo lib needs to.

File: server/controllers/connection-status.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            .then((item) => {
+                res.json({ item });
+            })
+            .catch((err) => {
+                res.status(500).json({ err });
+            });
+    },
+};
+
+//ConnectionStatusController.getConnectionStatus('577228983');
```
Comment (SaxyPandaBear) [inline server/controllers/connection-status.js:43]: clean up this comment

File: server/lib/dynamoDao.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+AWS.config.update({
+    region: 'us-east-1',
+    endpoint: 'http://localhost:8000',
+});
+var docClient = new AWS.DynamoDB.DocumentClient();
+function queryDynamoByChannel(channelId) {
+    const params = {
+        TableName: 'twitch-sessions',
+        Key: {
+            channelId: { S: `channel${channelId}` },
```
Comment (SaxyPandaBear) [inline server/lib/dynamoDao.js:16]: Why do we need to prepend `channel` to the ID? Is there a reason we need to store the channel id with that prefix? It's baggage for no reason. The name of the field tells us what the purpose is. We don't need to transform the value we get from Twitch just to prepend a constant `"channel"` to it.

**Comment by SaxyPandaBear** [comment]
Also, it looks like in the AWS docs that capitalized names are used for fields. Slight nitpick, but let's follow that guideline. Doesn't need to be in this PR if it can be cleaned up in a followup PR.

**Comment by ggenya132** [comment]
thanks you for the comments 🙏

**Pull request merged**